report_thesis/src/sections/introduction.tex

```diff
@@ -38,7 +38,11 @@ \section{Introduction}\label{sec:introduction}
 However, there remains considerable uncertainty about which machine learning techniques best predict the composition of major oxides in Martian geological samples using \gls{libs} data.
 This underscores the importance of a detailed study into advanced machine learning models for improving predictions in these applications.
 
-\textit{In this work, we aim to investigate the application of advanced machine learning models to predict the composition of major oxides in Martian geological samples using \gls{libs} data.}
+Furthermore, it is clear that dimensionality reduction plays a pivotal role in addressing the challenges posed by the high-dimensional nature of the data.
```

Comment:
Probably need a cite here? And perhaps an explicit "as shown by" reference, since this otherwise feels a little tacked-on.